packages/provider-mongo/test/globalSetup.js

```diff
@@ -2,9 +2,17 @@
 
 import { MongoMemoryServer } from 'mongodb-memory-server'
 
-export default async () => {
+export const setup = async () => {
   globalThis.__mongoServer = await MongoMemoryServer.create()
   // Each test file has its own environment with its own `global` instance so
   // we need to use the process environment.
-  process.env.MONGO_URI = global.__mongoServer.getUri()
+  process.env.VITE_MONGO_URI = global.__mongoServer.getUri()
+}
+
+// https://nodkz.github.io/mongodb-memory-server/docs/guides/integration-examples/test-runners/
```

Comment:
Here's the updated, working link: https://typegoose.github.io/mongodb-memory-server/docs/guides/integration-examples/test-runners